spec/controllers/container_image_registry_controller_spec.rb

```diff
@@ -18,7 +18,7 @@
     get :show, :id => container_image_registry.id
     expect(response.status).to eq(200)
     expect(response.body).to_not be_empty
-    expect(assigns(:breadcrumbs)).to eq([{:name => "Container Image Registry",
+    expect(assigns(:breadcrumbs)).to eq([{:name => "Container Image Registries",
```

Comment:
> @dkorn better leave that aside and open another pr for that bug.

I can open an issue and a separate PR/add another commit for the small fix ("Container Image Registry" -> "Container Image Registries" + spec), or leave it the way it is. @simon3z what do you think?